c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -708,14 +708,17 @@ $(PACKAGE)/locale/$(PACKAGE)-$(L10N_SERVER_POSTFIX).pot: \
 	[ ! -f $@ ] || sed -i '/^"PO-Revision-Date: /d' $@
 	[ ! -f $@ ] || chmod go+r $@
 
-.PRECIOUS: $(PACKAGE)/locale/$(PACKAGE)-$(L10N_CLIENT_POSTFIX).pot
-$(PACKAGE)/locale/$(PACKAGE)-$(L10N_CLIENT_POSTFIX).pot: \
-		lingua-client.cfg \
+TRANSLATION_DEPENDENCIES = lingua-client.cfg \
```

Comment:
is there a relay deference between TRANSLATION_DEPENDENCIES and I18N_DEPENDENCIES?